index/scorch/snapshot_index.go

```diff
@@ -471,6 +468,7 @@ func (i *IndexSnapshot) Document(id string) (rv index.Document, err error) {
 	if err != nil {
 		return nil, err
 	}
+
 	if diskStatsAvailable {
```

Comment:
Flag not necessary?

Reply:
Yeah these were minor rebase discrepancies. I've fixed them